app/helpers/layout_helper.rb

```diff
@@ -493,6 +493,10 @@ def new_child_fields_template(form_builder, association, options = { })
     end
   end
 
+  def spinner
+    '<span class="spinner-placeholder" />'.html_safe
```

Comment:
ok, the size of the span is 0 as there is no text, hence its not visible.
consider adding something like:
```css
 background: image-url('spinner.gif') no-repeat top left;
 display: inline-block;
```